canonical-websites/tutorials.ubuntu.com
# Update to ./run v2.2.0

Update to latest ./run script.

Please also review the generator which has been updated for bower install
https://github.com/canonical-webteam/generator-canonical-webteam/pull/48

# QA
Run site as normal and all should remain well.
Check out the auto demo and it too, should remain well.
